client/gapic_metadata.json

```diff
@@ -55,6 +55,16 @@
                 "RepeatDataBodyInfo"
               ]
             },
+            "RepeatDataBodyPatch": {
```

Comment:
Nit: this will be updated in the following regen PR, but its fine to keep here too.